internal/command/init.go

```diff
@@ -63,6 +65,8 @@ func (c *InitCommand) Run(args []string) int {
 	cmdFlags.StringVar(&flagLockfile, "lockfile", "", "Set a dependency lockfile mode")
 	cmdFlags.BoolVar(&c.Meta.ignoreRemoteVersion, "ignore-remote-version", false, "continue even if remote and local Terraform versions are incompatible")
 	cmdFlags.StringVar(&testsDirectory, "test-directory", "tests", "test-directory")
+	cmdFlags.BoolVar(&flagJson, "json", false, "json")
+
 	cmdFlags.Usage = func() { c.Ui.Error(c.Help()) }
```

Comment:
Notice that few instances of `c.Ui.Error` still exists in peculiar cases, e.g 
- where errors are caught before the view struct is even instantiated, (lines#70)
- in cases where an error is appended to diagnostics & also printed with colourize

Any thoughts on this?

Reply:
yes, however I am now revising this to display exclusively json or human format depending on the view type